### ISSUE TITLE
Light regression on ad_sepulcher

### ISSUE THREAD

**Issue by ericwa** [ericwa/tyrutils-ericw#181]
The fix for https://github.com/ericwa/tyrutils-ericw/issues/172 is causing issues in ad_sepulcher:
![spasm0010](https://user-images.githubusercontent.com/239161/31051453-fef8916c-a625-11e7-98eb-bd7ad3305784.png)
![spasm0011](https://user-images.githubusercontent.com/239161/31051454-01fe8736-a626-11e7-81d8-6ddaa2830bd1.png)
![spasm0012](https://user-images.githubusercontent.com/239161/31051455-04c3acd0-a626-11e7-9516-7326bd307dee.png)
![spasm0013](https://user-images.githubusercontent.com/239161/31051456-0796d554-a626-11e7-8898-160a4888f43a.png)

```
Viewpos: (-2156 -2956 38) -2 313 0
Viewpos: (-425 -417 63) 0 279 0
```

**Comment by ericwa** [comment]
What it's supposed to look like:
![spasm0014](https://user-images.githubusercontent.com/239161/31051478-aa8acb76-a626-11e7-8741-bae916195e9b.png)
![spasm0015](https://user-images.githubusercontent.com/239161/31051479-ac5f890a-a626-11e7-9348-71aa84e4fe15.png)

**Comment by ericwa** [comment]
I think what is happening is:
- the brushes have phong shading enabled.
- there is a point light slightly behind the wall surface in the window recess.
- The sample points on the wall still have line-of-sight to the point light.

The change in 26c5f65f4eb97ba5730443002196ec0a4aed8941 would see the "curved" flag set on the wall surface, even though it's not smoothed with the circular window trim, because phong is enabled per-brush.

**Comment by ericwa** [comment]
Fixed by 0a8a25b95cae727dfb534386615392a7833abd07 (reverting 26c5f)
